### ISSUE TITLE
AutoTSEstimator to support customized loss function and metric 

### ISSUE THREAD

**Issue by shupingyeh** [intel-analytics/BigDL#4035]
Hi,

Can AutoTSEstimator support customized loss function and metric for AutoML tuning?

**Comment by yushan111** [comment]
Sorry that we haven't supported customized loss and metric for now.

**Comment by TheaperDeng** [comment]
Customized loss is supported. https://bigdl.readthedocs.io/en/latest/doc/PythonAPI/Chronos/autotsestimator.html. 
String, pytorch/tf.keras loss instance or pytorch loss creator function are all supported

If customized metric is really needed we can add the support.

**Comment by TheaperDeng** [comment]
The latest nightly build has supported this feature (customized metric). https://pypi.org/project/bigdl-chronos/2.1.0b20220228/

@liangs6212 could you have some thorough UTs on this?

**Comment by shupingyeh** [comment]
Thank you very much!

**Comment by liangs6212** [comment]
> The latest nightly build has supported this feature (customized metric). https://pypi.org/project/bigdl-chronos/2.1.0b20220228/
> 
> @liangs6212 could you have some thorough UTs on this?

ok, I'll add some relevant UTs.
